scripts/installer.sh

```diff
@@ -400,7 +400,8 @@ main() {
 		freebsd)
 			if [ "$VERSION" != "12" ] && \
 			   [ "$VERSION" != "13" ] && \
-			   [ "$VERSION" != "14" ]
+			   [ "$VERSION" != "14" ] && \
+      	   		   [ "$VERSION" != "15" ]
```

Comment:
Small indentation adjustment
```suggestion
			   [ "$VERSION" != "15" ]
```